src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -160,7 +163,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('values', help="The task values/parameters file path relative to the source context.")
 
         # common to DockerBuildStep, FileTaskStep and RunTaskStep
-        c.argument('context_path', options_list=['--context', '-c'], help="The full URL to the source code repository (Requires '.git' suffix for a github repo).")
+        c.argument('context_path', options_list=['--context', '-c'], help="The full URL to the source code repository (Requires '.git' suffix for a github repo). /dev/null will be its value for a contextless task.")
```

Comment:
nit: might wanna rename based on what we are keeping
`contextless` vs `without a context path`